Gifski/Base.lproj/MainMenu.xib

```diff
@@ -68,12 +63,7 @@
                         <items>
                             <menuItem title="Open…" keyEquivalent="o" id="IAo-SY-fd9">
                                 <connections>
-                                    <action selector="openDocument:" target="-1" id="bVn-NM-KNZ"/>
-                                    <binding destination="Voe-Tx-rLC" name="enabled" keyPath="self.isRunning" id="zW7-dU-Zy9">
```

Comment:
Here, instead of `self.isRunning`. Is it possible to use `mainWindowController.isRunning`?

Reply:
Not sure. Couldn't find a way to hook this up in IB. Can try some more, maybe editing the XML will work, but that feels a little hacky..

Would you prefer to keep the window in MainMenu.xib, linked to app delegate and do this?
I guess we could do `MainWindowController(window window!)`

We could probably add a custom initialiser that sets up the window programmatically in MainWindowController too.

Reply:
I was thinking we just override `init` in the window controller and create the window there. I don't see the point of having a XIB file for this when we're already creating the window programmatically.

Reply:
I'm fine with using `validateMenuItem` for now. Though I'm surprised there's no good way to bind to values in an `NSWindowController` from `MainMenu.xib`. Seems like an obvious need.

Reply:
@sindresorhus Will have a look at setting up the window programmatically.

Do you know what style masks I should use?
```swift
let window = NSWindow(
  contentRect: rect,
  styleMask: [.titled, .fullSizeContentView],
  backing: .buffered,
  defer: false
)
```

Reply:
```swift
styleMask: [.titled, .closable, .fullSizeContentView]
```